tests/virtualenv_tools_test.py

```diff
@@ -207,3 +208,191 @@ def test_not_a_virtualenv_missing_site_packages(fake_venv, capsys):
         fake_venv, fake_venv.join('lib/python2.7/site-packages'),
     )
     assert out == expected
+
+# -------- BEGIN Temp PyPy support Tests ------------
+# After base functionality for pypy venv movement is reviewed
+# these can be refactored in a way similar to how it was done in
+# https://github.com/Yelp/virtualenv-tools/pull/11 to reduce duplication
+
+
+skip_test_if_requires_pypy3 = pytest.mark.skipif(
+    sys.version_info >= (3, 3),
+    reason='pypy3 is not supported.'
+)
```

Comment:
You're skipping these tests for cpython3.6 as well.
Doesn't seem right.

What goes wrong if we delete this thing?

- [ ] fixed